snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/IdRequest.java

```diff
@@ -119,6 +120,7 @@ public R execute(final C context) {
 						}
 						
 						final Multimap<String, BaseSnomedComponentCreateRequest> requestsByNamespace = FluentIterable.from(categoryRequests)
+								.filter(BaseSnomedComponentCreateRequest.class)
```

Comment:
It is necessary to use `BaseSnomedComponentCreateRequest` because we are using the `BaseSnomedComponentCreateRequest.setIdGenerationStrategy()` method later.